src/components/molecules/Visualizer/Engine/ref.ts

```diff
@@ -48,6 +48,19 @@ export type MouseEventHandles = {
   onWheel: (fn: MouseEvents["wheel"]) => void;
 };
 
+type ClockEventHandler<R = void> = (cb: (clock: Clock) => void) => R;
+export type Clock = {
+  startTime: Date;
+  stopTime: Date;
+  currentTime: Date;
+  tick: () => Date;
+  shouldAnimate: boolean;
```

Comment:
```suggestion
  isPlaying: boolean;
```

I think it's easy to understand

Reply:
I fixed in [refactor: rename from shouldAnimate to isPlaying](https://github.com/reearth/reearth-web/pull/285/commits/89c396f37e8b076abf7bb3ab24b578ab612c48e6)